launcher/ui/pages/global/JavaPage.cpp

```diff
@@ -134,6 +152,14 @@ void JavaPage::on_javaDetectBtn_clicked()
     if (vselect.result() == QDialog::Accepted && vselect.selectedVersion()) {
         java = std::dynamic_pointer_cast<JavaInstall>(vselect.selectedVersion());
         ui->javaPathTextBox->setText(java->path);
+        if (!java->is_64bit && APPLICATION->settings()->get("MaxMemAlloc").toInt() > 2048) {
+            CustomMessageBox::selectable(this, tr("Confirm Selection"),
+                                         tr("You selected an x86 java version.\n"
```

Comment:
```suggestion
                                         tr("You selected a 32 bit java version.\n"
```

I think 32 bit is more clear than x86 here